### PR TITLE
This is a practice commit

### PR DESCRIPTION
**Pull request by guineveresaenger**
This is a paragraph describing your intent and the background for this pull request

### CODE REVIEW

**Comment by castrojo** [comment]
Hello Kubecon!

**Comment by jberkus** [comment]
Is there an issue associated with this PR?

File: contributors/new-contributor-playground/new-contributors.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Hello everyone! Please feel free to talk amongst yourselves or ask questions if you need help
```
Comment (jberkus) [inline contributors/new-contributor-playground/new-contributors.md]: This might be better as two lines.

**Pull request merged**